cmd/konflux-apply/main.go

```diff
@@ -21,31 +16,13 @@ func main() {
 	ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt, os.Kill)
 	defer cancel()
 
-	var (
-		inputConfig string
-		konfluxDir  string
-		excludes    []string
-	)
-
-	defaultExcludes := []string{
-		fmt.Sprintf(".*%s.*", konfluxgen.ReleasePlanAdmissionsDirectoryName),
-	}
-
-	pflag.StringVar(&inputConfig, "config", filepath.Join("config"), "Specify repositories config")
-	pflag.StringVar(&konfluxDir, "konflux-dir", ".konflux", "Konflux directory containing applications, components, etc")
-	pflag.StringArrayVar(&excludes, "exclude", defaultExcludes, "Regex patterns of files or directories to exclude from apply")
-
-	pflag.Parse()
-
-	excludeRegex, err := util.ToRegexp(excludes)
-	if err != nil {
-		log.Fatal("failed to parse excludes regex: ", err)
-	}
+	inputConfig := flag.String("config", filepath.Join("config"), "Specify repositories config")
+	konfluxDir := flag.String("konflux-dir", ".konflux", "Konflux directory containing applications, components, etc")
```

Comment:
```suggestion
	konfluxDir := flag.String("konflux-dir", filepath.Join(".konflux", konfluxgen.ApplicationsDirectoryName), "Konflux directory containing applications, components, etc")
```

Reply:
Had it in https://github.com/openshift-knative/hack/blob/000150557f189101a99df727e1d7b75835a264f0/pkg/konfluxapply/konflux_apply.go#L95, but this is clearer.
Updated it